### ISSUE TITLE
Enzyme: identifier `ReactClass`. Could not resolve name

### ISSUE THREAD

**Issue by ml9951** [flowtype/flow-typed#1158]
I'm getting the following error with the Enzyme type defs:

```
flow-typed/npm/enzyme_v2.3.x.js:7
  7:   declare type EnzymeSelector = string | ReactClass<any> | Object;
                                              ^^^^^^^^^^^^^^^ identifier `ReactClass`. Could not resolve name
```

I'm using `"flow-bin": "^0.53.1",`.

**Comment by joewalker** [comment]
This works for me: ReactClass -> React$ComponentType

**Comment by rosskevin** [comment]
This workaround worked better for us:
```
declare type EnzymeSelector = string | React$Element<*> | Object;
```

I noted it in the open PR.

**Comment by GAntoine** [comment]
Closing as this was fixed in #1161.
